crates/xline-client/src/clients/lock.rs

```diff
@@ -84,6 +73,10 @@ impl LockClient {
     ///
     /// This function will return an error if the inner CURP client encountered a propose failure
     ///
+    /// # Panics
+    ///
+    /// Panic if the given `LockRequest.inner.lease` less than or equal to 0
+    ///
```

Comment:
Could it be better to return an error than panic?

Reply:
We don't need to panic or return an error when issue #684  is closed.